File: Service_eHospital_DongNai_A_NSTL/Implement/NS_RP_FORMULARSService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_RP_FORMULARSService : BaseService<NS_RP_FORMULARS, INS_RP_FORMULARSRepository>
    , INS_RP_FORMULARSService
    {
    private readonly INS_RP_FORMULARSRepository _NS_RP_FORMULARSRepository;
    public NS_RP_FORMULARSService(INS_RP_FORMULARSRepository NS_RP_FORMULARSRepository) : base(NS_RP_FORMULARSRepository)
    {
    _NS_RP_FORMULARSRepository = NS_RP_FORMULARSRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_RP_FORMULARS.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_RP_FORMULARS } from './NS_RP_FORMULARS.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_RP_FORMULARSService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'REPORTCODE', 'NAME', 'CONTROLNAME', 'VALUE'];

    List: NS_RP_FORMULARS[] | undefined;
    ListFilter: NS_RP_FORMULARS[] | undefined;
    FormData!: NS_RP_FORMULARS;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_RP_FORMULARS";
    }
}

